drivers/metal/rendering_shader_container_metal.mm

```diff
@@ -417,6 +417,8 @@
 							// VK_DESCRIPTOR_TYPE_STORAGE_TEXEL_BUFFER
 							primary.texture_type = MTLTextureTypeTextureBuffer;
 						} break;
+						case spv::DimTileImageDataEXT: {
+						} break;
```

Comment:
FYI @stuartcarnie 
We might want to actually handle this one, I didn't check what it's for.

Reply:
@akien-mga this is the correct fix, as we aren't using it, so we can ignore it for now.

It requires this extension https://registry.khronos.org/vulkan/specs/latest/man/html/VK_EXT_shader_tile_image.html